website/source/guides/advanced-scheduling/spread.html.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: "guides"
+page_title: "Spread"
+sidebar_current: "guides-advanced-scheduling"
+description: |-
+    The spread stanza allows operators to increase failure tolerance in their Nomad clusters by allowing them to distribute their workloads in a customized way based on attributes or client metadata.
```

Comment:
Same as above. the description for this should say its a guide or walkthrough about the spread feature.